starters/adapters/cloudflare-pages/package.json

```diff
@@ -2,7 +2,7 @@
   "description": "Cloudflare Pages",
   "scripts": {
     "build.server": "vite build -c adapters/cloudflare-pages/vite.config.ts",
-    "deploy": "wrangler pages dev ./dist"
+    "deploy": "wrangler pages publish ./dist --branch preview"
```

Comment:
Ugh! The actual fix is to remove option this altogether. That will result in the project using the current git branch. Then by default if you are on the `main` branch (and `main` has been selected as the production branch) this will deploy a production site. Otherwise, on any other branch it will deploy a preview site.

See https://developers.cloudflare.com/pages/platform/branch-build-controls/#production-branch-control

Reply:
yes I did mention this above: https://github.com/BuilderIO/qwik/pull/2422#issuecomment-1355326380

Reply:
Sorry I posted this in the wrong PR!!